app/serializers/gym_serializer.rb

```diff
@@ -1,33 +1,24 @@
 class GymSerializer
-  def self.format_gyms(gyms)
-    {
-      data:
-        gyms.map do |gym|
-          {
-            id: gym.yelp_gym_id,
-            type: 'gym',
-            attributes: {
-              name: gym.name,
-              address: gym.address,
-              phone: gym.phone
-            }
-          }
-        end
-    }
-  end
+  class << self
+    def format_gyms(gyms)
+      { data: gyms.map { |gym| reformat_gym(gym) } }
+    end
+
+    def format_gym(gym)
+      { data: reformat_gym(gym) }
+    end
```

Comment:
Nice!  I like this refactor.

Reply:
Thanks!!  Felt like I could give it a DRY as I was making updates